helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -375,13 +375,15 @@ public void removeDeploymentGroup(final String name) throws DeploymentGroupDoesN
     final ZooKeeperClient client = provider.get("removeDeploymentGroup");
     try {
       client.ensurePath(Paths.configDeploymentGroups());
-      client.delete(Paths.configDeploymentGroup(name));
+
```

Comment:
The problem mbetter had was that `/config/deployment-group/<name>` didn't exist but `/status/deployment-group/<name>` and `/status/deployment-group/hosts/<name>`. He couldn't create or remove his DG.

This change will remove all relevant DG ZK nodes even if the `/config/deployment-group/<name>` path doesn't exist.